CeGameApp-SPA/src/app/homepage/homepage.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-homepage',
+  templateUrl: './homepage.component.html',
+  styleUrls: ['./homepage.component.css']
+})
+export class HomepageComponent implements OnInit {
+
+  constructor() { }
```

Comment:
consider checking your classes and make sure you are not using methods that are doing nothing.